.github/workflows/tag_and_release_protos.yml

```diff
@@ -57,7 +57,14 @@ jobs:
       image: ghcr.io/viamrobotics/canon:amd64
     strategy:
       matrix:
-        repo: ['viamrobotics/rdk','viamrobotics/app','viamrobotics/python-sdk','viamrobotics/rust-sdk','viamrobotics/rust-utils','viamrobotics/viam-cpp-sdk']
+        repo:
+          - viamrobotics/rdk
+          - viamrobotics/app
+          - viamrobotics/python-sdk
+          - viamrobotics/rust-sdk
+          - viamrobotics/rust-utils
+          - viamrobotics/viam-cpp-sdk
+          - viamrobotics/viam-typescript-sdk
```

Comment:
i think this is equivalent to the bracket notation?

Reply:
is the best way to test just fire off this action? i guess it will just open several PRs that we close...

Reply:
it worked! [this run](https://github.com/viamrobotics/api/actions/runs/4558097336/jobs/8040569158) opened [this PR](https://github.com/viamrobotics/viam-typescript-sdk/pull/40)